msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -1028,9 +1028,7 @@ You've completed the tutorial to build a tab app with Blazor.
 
 > [!NOTE]
 >
-> * Migrating your configurable tab to static tab is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
```

Comment:
We shouldn't remove this section yet since the feature is not yet available in R4/General ring.

Reply:
These docs are specifically about publishing updated docs once the feature reaches general availability 😉 so, let's be sure to remove this section for the GA docs.